set.mm

```diff
@@ -7233,12 +7233,41 @@ _praeclarum theorema_ (splendid theorem).  (Contributed by NM,
 
 $( Relation of conjunction to bi-conditional. $)
 
-  ${
-    biadan2.1 $e |- ( ph -> ps ) $.
-    biadan2.2 $e |- ( ps -> ( ph <-> ch ) ) $.
-    $( Add a conjunction to an equivalence.  (Contributed by Jeff Madsen,
-       20-Jun-2011.) $)
-    biadan2 $p |- ( ph <-> ( ps /\ ch ) ) $=
+  $( An implication is equivalent to the equivalence of some implied
+     equivalence and some other equivalence.  A utility lemma as illustrated in
+     ~ biadanii and ~ elelb .  (Contributed by BJ, 4-Mar-2023.) $)
+  biadan $p |- ( ( ph -> ps ) <->
+                   ( ( ps -> ( ph <-> ch ) ) <-> ( ph <-> ( ps /\ ch ) ) ) ) $=
+    ( wi wa wb pm5.32 bicom bitri biass mpbir pm4.71r bitr4i bibi2i 3bitri mpbi
+    ) ABDZABCEZFZBACFDZFZTSFTRFZQAFZFZQUAFZUBBAEZUCUBUFFTRUFFZFTUFRFUGBACGUFRHI
+    TRUFJKUCUFFQAUFFFABLQAUFJKMUDUCUBFQAUBFZFUEUBUCHQAUBJUHUAQUHARTFZFUAUBUIATR
+    HNARTJMNOPSTHI $.
+
+  ${
+    biadani.1 $e |- ( ph -> ps ) $.
+    $( Inference associated with ~ biadan .  (Contributed by BJ,
+       4-Mar-2023.) $)
+    biadani $p |- ( ( ps -> ( ph <-> ch ) ) <-> ( ph <-> ( ps /\ ch ) ) ) $=
+      ( wi wb wa biadan mpbi ) ABEBACFEABCGFFDABCHI $.
+
+    $( Alternate proof of ~ biadani not using ~ biadan .  (Contributed by BJ,
+       4-Mar-2023.)  (Proof modification is discouraged.)
+       (New usage is discouraged.) $)
+    biadaniALT $p |- ( ( ps -> ( ph <-> ch ) ) <-> ( ph <-> ( ps /\ ch ) ) ) $=
+      ( wb wi wa pm5.32 pm4.71ri bibi1i bitr4i ) BACEFBAGZBCGZEAMEBACHALMABDIJK
+      $.
+
+    biadanii.2 $e |- ( ps -> ( ph <-> ch ) ) $.
+    $( Inference associated with ~ biadani .  Add a conjunction to an
+       equivalence.  (Contributed by Jeff Madsen, 20-Jun-2011.)  (Proof
+       shortened by BJ, 4-Mar-2023.) $)
+    biadanii $p |- ( ph <-> ( ps /\ ch ) ) $=
+      ( wb wi wa biadani mpbi ) BACFGABCHFEABCDIJ $.
+
+    $( Obsolete proof of ~ biadanii as of 4-Mar-2023.  (Contributed by Jeff
+       Madsen, 20-Jun-2011.)  (Proof modification is discouraged.)
+       (New usage is discouraged.) $)
+    biadaniiOLD $p |- ( ph <-> ( ps /\ ch ) ) $=
```

Comment:
Only formalism: since the old theorem was ~biadan2, this deprecated theorem should be called "biadan2OLD".

Reply:
done.